### PR TITLE
61/require loc

### PR DESCRIPTION
**Pull request by kftang**

### CODE REVIEW

File: routes/foodtrucks_test.go [hunk 1]
```diff
@@ -95,3 +102,73 @@ func TestFoodTruckGetValid(t *testing.T) {
...
+	if rr.Code != expected {
+		t.Errorf("adding invalid food truck expected status code of %v, but got %v", expected, rr.Code)
+	}
+}
+
+func TestFoodTruckPostValid(t *testing.T) {
+	tests.ClearDB()
+
+	name := "test cafe"
+	address := "1 direction st"
```
Comment (janinebar) [inline routes/foodtrucks_test.go:358]: Nice

File: routes/foodtrucks_test.go [hunk 2]
```diff
@@ -95,3 +102,73 @@ func TestFoodTruckGetValid(t *testing.T) {
...
+	handler := http.HandlerFunc(PostFoodTrucksHandler)
+	handler.ServeHTTP(rr, req)
+
+	expected := http.StatusUnauthorized
+	if rr.Code != expected {
+		t.Errorf("adding food truck while unauthorized expected status code of %v, but got %v", expected, rr.Code)
+	}
+}
+
+func TestFoodTruckPostInvalid(t *testing.T) {
```
Comment (janinebar) [inline routes/foodtrucks_test.go]: Test single food truck or multiple food trucks? Look at func name and endpoint, make sure its clear

File: routes/foodtrucks_test.go [hunk 3]
```diff
@@ -95,3 +102,73 @@ func TestFoodTruckGetValid(t *testing.T) {
 		t.Errorf("expected food truck with id test, but got %v", foodTruck)
 	}
 }
+
+func TestFoodTruckPostUnauthorized(t *testing.T) {
```
Comment (janinebar) [inline routes/foodtrucks_test.go]: Test single food truck or multiple food trucks? Look at func name and endpoint, make sure its clear

**Review by janinebar — CHANGES_REQUESTED** [review]

File: routes/foodtrucks_test.go (hunk 2 above)
Comment (kftang) [reply routes/foodtrucks_test.go]: u rite

File: routes/foodtrucks_test.go (hunk 3 above)
Comment (kftang) [reply routes/foodtrucks_test.go]: fixed

**Review by janinebar — APPROVED** [review]
tnx 4 da fix

**Pull request merged**